### PR TITLE
Shorten the classpath in bin/start-handle-server to address inconsistent ClassNotFound

### PR DESCRIPTION
**Pull request by mwoodiupui**
## References
* Related to #8507

## Description
Get the classpath from bin/dspace, which uses a special JRE feature that means "all JARs in this directory".
Note that the same issue exists on Windows.  Another patch would be welcome.

## Instructions for Reviewers
List of changes in this PR:
* call `bin/dspace classpath` instead of duplicating (old, replaced) code.
* Corrected comments, tidied up whitespace, removed unnecessary differences between `start-handle-server` and `dspace`.

## Checklist
- [x] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & integration tests). Exceptions may be made if previously agreed upon.
- [x] My PR passes Checkstyle validation based on the [Code Style Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Style+Guide).
- [x] My PR includes Javadoc for _all new (or modified) public methods and classes_. It also includes Javadoc for large or complex private methods.
- [x] My PR passes all tests and includes new/updated Unit or Integration Tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [x] If my PR includes new, third-party dependencies (in any `pom.xml`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.
- [x] If my PR modifies the REST API, I've linked to the REST Contract page (or open PR) related to this change.

### CODE REVIEW

File: dspace/bin/start-handle-server [hunk 1]
```diff
@@ -21,19 +21,23 @@ HANDLEDIR=`$BINDIR/dspace dsprop --property handle.dir`
...
 #   Any existing classpath
-#   The JARs (WEB-INF/lib/*.jar)
-#   The WEB-INF/classes directory
-FULLPATH=$CLASSPATH:$JARS:$DSPACEDIR/config
+#   The JARs (lib/*.jar)
+#   The configuration directory
+if [ "$CLASSPATH" = "" ]; then
+  FULLPATH=$JARS:$DSPACEDIR/config
+else
+  FULLPATH=$CLASSPATH:$JARS:$DSPACEDIR/config
```
Comment (tdonohue) [inline dspace/bin/start-handle-server:34]: Just reading the code, I *think* we can drop the `/config` directory from both the `if` and `else`, as it's already included by `./dspace classpath`.

In fact, we might be able to drop this entire `if/else` and ONLY call `./dspace classpath`.  But, that might need more testing

Comment (mwoodiupui) [reply dspace/bin/start-handle-server:34]: `config/`:  you're right -- we can *and should* drop the additional copy.

`if/else`:  I consider this a sort of anti-bugging.  Without the `if` we may end up with an empty leading path element if CLASSPATH is undefined, which is common enough.  It's unlikely that a JRE would ever have serious trouble with that, but it's code out of our control....

Comment (mwoodiupui) [reply dspace/bin/start-handle-server:34]: Actually `bin/dspace classpath` is already doing everything we need, so I just got rid of $JARS and the whole path-construction section.

**Review by tdonohue — APPROVED** [review]
👍 Thanks @mwoodiupui !  I've tested this locally and the changes to `./start-handle-server` work well for me.  Also retested the `./dspace` CLI script though its changes were minimal.

**Pull request merged**